Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2114,18 +2201,18 @@ private DataType getExpressionDefResultType(ExpressionDef expressionDef) {
             return expressionDef.getResultType();
         }
 
-        // If the current expression context is patient, a reference to a population context expression will indicate a full
+        // If the current expression context is specific, a reference to an unspecified context expression will indicate a full
```

Comment:
Only a comment, but probably should replace `unspecified` with `unfiltered`

Reply:
Agreed, corrected